jbmc/regression/jbmc/simplify-classid-of-interface/test.desc

```diff
@@ -0,0 +1,15 @@
+CORE
+Test.class
+--function Test.main
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+^Passing problem to
+^warning: ignoring
+--
+The "Passing problem to..." line is a sign that symex couldn't prove the
```

Comment:
What about "remaining VCCs: 0" ?

Reply:
Could you match the simplified expression on the `--program-only` output?

Reply:
Added a test checking the expected expression is in the --show-vccs output